public/dist/data/helper-scripts/locations.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * from the tourism site
+ */
+
+var locationsElms = $('#townName').find('option').toArray();
```

Comment:
Global vars - please namespace or iife this.

Reply:
done